crates/tools/src/main.rs

```diff
@@ -163,3 +160,11 @@ fn run(cmdline: &'static str, dir: &str) -> Result<()> {
     }
     Ok(())
 }
+
+fn run_rustfmt() -> Result<()> {
+    // Use beta toolchain for 2018 edition.
+    run("rustup install beta", ".")?;
```

Comment:
Let's pin this to a specific beta, `rustup install beta-2018-10-30`

Reply:
How about we have a const value like:
```rust
const TOOLCHAIN: &str = "beta-2018-10-30";
```
and use it where we need?

Reply:
:+1: